gateway/ln-gateway/proto/gateway_lnrpc.proto

```diff
@@ -176,3 +190,37 @@ message GetRouteHintsResponse {
   // The route hints to the associated lightning node
   repeated RouteHint route_hints = 1;
 }
+
+message ConnectToPeerRequest {
+  // The public key of the node we're connecting to.
+  string pubkey = 1;
+
+  // The host address of the node we're connecting to.
+  string host = 2;
+}
+
+message GetFundingAddressResponse {
+  // An address belonging to the lightning node's on-chain wallet.
+  string address = 1;
+}
+
+message OpenChannelRequest {
+  // The public key of the node we're opening a channel to.
+  string pubkey = 1;
+
+  // The capacity of the channel, in sats.
+  uint64 channel_size_sats = 2;
+
+  // The amount of sats that should be pushed to the
+  // counterparty once the channel is opened.
+  uint64 push_amount_sats = 3;
+}
+
+message WaitForChainSyncRequest {
+  // The height that the lightning node should have fully processed before returning.
+  uint32 block_height = 1;
+
+  uint32 max_retries = 2;
```

Comment:
Can remove this if we refactor to just get the block height

Reply:
Leaving this for now while we figure out if we're even going to keep `WaitForChainSync`